packages/deployments/contracts/contracts/messaging/interfaces/ISpokeConnector.sol

```diff
@@ -1,29 +0,0 @@
-// SPDX-License-Identifier: MIT OR Apache-2.0
-pragma solidity 0.8.15;
-
-import {IConnector} from "./IConnector.sol";
-
-interface ISpokeConnector is IConnector {
```

Comment:
do you want to keep anything about the interface here..?

Reply:
I just deleted this because we weren't using it anywhere - we weren't even implementing it anymore in latest main. The only thing I used it for was for the Proof struct and tests, and Proof struct ended up getting moved to SpokeConnector anyway (and the tests worked fine with just using the contract directly as the interface).

Lmk if you think we should keep it tho... I can reimplement it, too if needed

Reply:
fair -- i thought it was nice to have those in one place but lets ditch it if we weren't even using it